streams/src/test/java/org/apache/kafka/streams/perf/SimpleBenchmark.java

```diff
@@ -717,7 +717,7 @@ public void close() {
 
 
     private double megaBytePerSec(long time, int numRecords, int recordSizeBytes) {
-        return (double) (recordSizeBytes * numRecords / 1024 / 1024) / ((double) time / 1000);
+        return (double) ((long) recordSizeBytes * numRecords / 1024 / 1024) / ((double) time / 1000);
```

Comment:
Nice catch. Would the following be a little better?

return ((double) recordSizeBytes * numRecords / 1024 / 1024) / (time / 1000.0);

We'd do double instead of integer division. for the first part. Also, can we rename `megaBytePerSec` to `megabytesPerSec`? or `mbPerSec`?